sys/nginx/default.conf.template

```diff
@@ -27,12 +27,12 @@ server {
 
     root /application/public;
 
-    if ($request_method = POST) { set $skip_cache 1; }
-    if ($query_string != "") { set $skip_cache 1; }
+    if ($request_uri ~ "/dark-canary/") { set $skip_cache 1; }
 
     fastcgi_cache cached_badges;
+    fastcgi_cache_methods GET HEAD;
     fastcgi_cache_key "$request_method-$scheme://$host$request_uri";
```

Comment:
As a best practice always add a delimiter which is not part of the keys value ("@@" for example) among the parts of the cache keys .. this is necessary to avoid cache keys collisions in rare but possible cases (this could be not the case, but better to follow best practices even when not strictly necessary ;) ).

I'm not sure the `scheme` has to be a cache key anyway 🤔

Reply:
fixed